### DISCUSSION TITLE
ScriptReference/Mesh

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#309] [Notes]
# ScriptReference/Mesh

https://docs.unity3d.com/ScriptReference/Mesh.html

**Comment by docscomments** [comment]
**Comment by `@Ragzouken`:**  

if you are updating vertices without updating the topology, absolutely remember to call [`RecalculateBounds`](https://docs.unity3d.com/ScriptReference/Mesh.RecalculateBounds.html) or you may find your mesh vanishing (i.e culled) when you look at it the wrong way!

**Comment by vertxxyz** [comment]
If you are creating a Mesh via `new Mesh` it should should be [Destroyed](https://docs.unity3d.com/ScriptReference/Object.Destroy.html) when you are done with it. As with materials and other native objects, they will otherwise only be cleaned up when [`Resources.UnloadUnusedAssets`](https://docs.unity3d.com/ScriptReference/Resources.UnloadUnusedAssets.html) runs.

**Comment by andyg336** [comment]
https://github.com/Unity-Technologies/MeshApiExamples Is omitted from the help documentation which it really shouldn't be
